spec/controllers/articles_controller_spec.rb

```diff
@@ -122,7 +122,12 @@
         article1 = create(:article)
         article2 = create(:article)
         get :index
-        expect(assigns(:articles)).to eq([article2, article1])
+
+        assigned_articles = assigns(:articles)
+        binding.pry
```

Comment:
need to remove this debug output

Reply:
This was removed in an unpushed commit, thanks for catching.